webpack.config.js

```diff
@@ -52,15 +48,7 @@ function generateWebpackConfigForCanister(name, info) {
       filename: "index.js",
       path: path.join(outputRoot, "assets"),
```

Comment:
Maybe I did something wrong, but I got this error when I tried opening the entrypoint in a browser (I did at least clear my cache):
```
An error happened:
http://localhost:8000/bootstrap.js:2:43236
asyncFunctionResume@[native code]
[native code]
promiseReactionJobWithoutPromise@[native code]
promiseReactionJob@[native code]
```

There are some more differences in the webpack.config.js for new projects that I don't see reflected here (though I don't know if they are required):
- [entry using frontend.entrypoint](https://github.com/dfinity-lab/sdk/blob/master/src/dfx/assets/new_project_node_files/webpack.config.js#L40)
- [output into a "dist" directory](https://github.com/dfinity-lab/sdk/blob/master/src/dfx/assets/new_project_node_files/webpack.config.js#L52)
- a corresponding assets `source` that looks like this:
```
      "source": [
        "src/sample_assets/assets",
        "dist/sample_assets/"
      ],
```

Reply:
Yes, I got the same error when pulling from this update. Interestingly, it worked without the change for me.

Reply:
Yep, it worked for me too when we had `"canisters/linkedup_assets/assets"` included..

Reply:
Okay.  What's happening is `webpack.config.js` is [writing its output](https://github.com/dfinity-lab/linkedup/blob/master/webpack.config.js#L53) into `canisters/linkedup_assets/assets`.

So it will work to put `canisters/linkedup_assets/assets` back into `sources`, because that's where webpack is putting the files, but that means dfx will use `canisters/linkedup_assets/assets` as both a source and as a destination when "building" the assets.  That might cause problems the second or third time a build is run (or it might "just work").

My understanding of how newly-created projects manage this:
- webpack.config.js writes its output [into a dist directory](https://github.com/dfinity-lab/sdk/blob/master/src/dfx/assets/new_project_node_files/webpack.config.js#L52)
- `dfx new` [adds this dist directory as a source](https://github.com/dfinity-lab/sdk/blob/master/src/dfx/src/commands/new.rs#L292)